docs/12-ContinuousIntegration.md

```diff
@@ -119,8 +119,28 @@ Travis doesn't provide visualization for XML or HTML reports so you can't view r
 
 ## GitLab
 
-[Work In Progress]
+If a file ```.gitlab-ci.yml``` exists in the root of the git repository, gitlab will run a pipeline each time you push to the gitlab server. The file configures the docker image that will be called. Below is a sample which loads a php7 docker image, clones your files, installs composer dependencies, runs the built-in php webserver and finally runs codeception:
+
+```
+image: mychiara/php7
```

Comment:
I have tried the codeception/codeception image. However it does not contain any shell. Even before the ```before_script``` section is run, I get this error:
```
  [Codeception\Exception\ConfigurationException]                
  Your configuration file `if [ -x /usr/local/bin/bash ]; then  
  	exec /usr/local/bin/bash                                     
  elif [ -x /usr/bin/bash ]; then                               
  	exec /usr/bin/bash                                           
  elif [ -x /bin/bash ]; then                                   
  	exec /bin/bash                                               
  elif [ -x /usr/local/bin/sh ]; then                           
  	exec /usr/local/bin/sh                                       
  elif [ -x /usr/bin/sh ]; then                                 
  	exec /usr/bin/sh                                             
  elif [ -x /bin/sh ]; then                                     
  	exec /bin/sh                                                 
  else                                                          
  	echo shell not found                                         
  	exit 1                                                       
  fi                                                            
  ` could not be found.                                         
```
So I guess I will use the php:7.0 image.